src/components/AnonymousChat/child/ChatReplyPreview.tsx

```diff
@@ -42,15 +44,31 @@ const ChatReplyPreview = ({type}: ChatReplyPreviewProps) => {
         activeOpacity={replyPreview?.id ? 0.75 : 1}
         onPress={handleTap}
         style={textContainerStyle}>
-        <View style={styles.chatTitleContainer}>
-          <Text style={styles.userText}>{replyPreview?.user?.username}</Text>
-          <View style={styles.dot} />
-          <Text style={styles.timeText}>{calculateTime(replyPreview?.updated_at, true)}</Text>
-        </View>
+        <View>
+          <View style={styles.chatTitleContainer}>
+            <Text style={styles.userText}>{replyPreview?.user?.username}</Text>
+            <View style={styles.dot} />
+            <Text style={styles.timeText}>{calculateTime(replyPreview?.updated_at, true)}</Text>
+          </View>
 
-        <Text style={messageStyle} numberOfLines={1}>
-          {replyPreview?.message ?? ''}
-        </Text>
+          <Text style={messageStyle} numberOfLines={1}>
+            {replyPreview?.attachments.length > 0 ? 'Photo' : `${replyPreview?.message ?? ''}`}
+          </Text>
+        </View>
+        {replyPreview?.attachments.length > 0 && (
+          <View style={styles.mediaContainer}>
+            <FastImage
+              source={{uri: replyPreview?.attachments[0]?.asset_url}}
+              style={styles.mediaImage}
+              resizeMode="cover"
+            />
+            {replyPreview?.attachments[0].video_path && (
+              <View style={styles.mediaOverlay}>
+                <IconVideoPlay width={24} height={24} />
+              </View>
+            )}
+          </View>
+        )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This code block has been significantly modified to support attachments in chat messages. The logic seems correct: if there are attachments, it displays "Photo" as the message text and renders the first attachment with `FastImage`. If the attachment is a video, an overlay with a play icon is shown. However, consider handling other types of attachments (like documents or audio files) and multiple attachments.
```diff
+         <View>
+           <View style={styles.chatTitleContainer}>
+             <Text style={styles.userText}>{replyPreview?.user?.username}</Text>
+             <View style={styles.dot} />
+             <Text style={styles.timeText}>{calculateTime(replyPreview?.updated_at, true)}</Text>
+           </View>
+  
+           <Text style={messageStyle} numberOfLines={1}>
+             {replyPreview?.attachments.length > 0 ? 'Photo' : `${replyPreview?.message ?? ''}`}
+           </Text>
+         </View>
+         {replyPreview?.attachments.length > 0 && (
+           <View style={styles.mediaContainer}>
+             <FastImage
+               source={{uri: replyPreview?.attachments[0]?.asset_url}}
+               style={styles.mediaImage}
+               resizeMode="cover"
+             />
+             {replyPreview?.attachments[0].video_path && (
+               <View style={styles.mediaOverlay}>
+                 <IconVideoPlay width={24} height={24} />
+               </View>
+             )}
+           </View>
+         )}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->